modules/common/src/main/MarkdownRender.scala

```diff
@@ -262,6 +271,15 @@ object MarkdownRender:
         .tag("/div")
         .unit
 
+    // Catches the site headers for https://github.com/lichess-org/lila/issues/11450
```

Comment:
Notice the change here as well.  You were correctly wrapping Node.getParent in an option to handle null, but then called get (which throws an exception on None).  I used fold to specify an explicit return value in case getParent ever misbehaves.

Reply:
That's a lot better yes. Note that `.fold(false)(f)` == `.exists(f)`